depecher/qml/pages/items/WritingItem.qml

```diff
@@ -285,7 +285,7 @@ Drawer {
                 }
                 IconButton {
                     id:stickerButton
-                    icon.source: "image://theme/icon-m-other"
+                    icon.source: "qrc:/icons/icon-m-sticker.png"
```

Comment:
Does it have some kind of license (same as [https://github.com/elros34/depecher/issues/4](url)) or you made it by yourself and don't care about it? I know depecher's license is not 100% clear but I don't want to make it even worse. BTW stickers are partially broken:)

Reply:
Both image are GPLV3

Reply:
My new sticker is broken with Depecher?

Reply:
qrc:/ means it expected to be in resource file. So either add it there and build depecher or for test you can just use full path to image without qrc:/ or image:/. Also please add author (and type of license) of that icon at least to README or some other file.

Reply:
oOoO you mean that the sticker is not working well at the moment, like the sticker get  freezen. 
The problem seems with RAM currently the depecher sticker load to much the RAM, we need separate the load by sections.

Reply:
I didn't notice any freeze. Actually I mean that you can't add new stickers unless you tap some sticker in conversation. There were some reports about it at openrepos so this is quite old bug.

Reply:
I you have lot sticker folder, then get freeze depend on how many RAM your device have, yes i agree that sticker only can added by  tap some sticker in conversation.